src/components/logoComponent/Logo.tsx

```diff
@@ -0,0 +1,30 @@
+import { cva } from 'class-variance-authority';
+import { twMerge } from 'tailwind-merge';
+
+const imageStyles = cva(['w-full', 'rounded'], {
```

Comment:
should we have cva in the same file or the seperate file along?. please give your suggestion @MohitPatidar29  @AnjaliRathore01  @mayanktamrkar